packages/core-forger/__tests__/client.test.js

```diff
@@ -45,7 +45,13 @@ describe('Client', () => {
     })
 
     describe('when the host is available', () => {
+      beforeEach(async () => {
```

Comment:
There is only 1 test in this `describe` block, `await client.__chooseHost()` should be called inside the `it` block.